frontend/src/components/Inputs/TextInput.tsx

```diff
@@ -0,0 +1,23 @@
+import "./Inputs.modules.scss";
+const TextInput = (props: {
+  name: any;
+  value: any;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  placeholder?: any;
```

Comment:
suggestion: avoid using `any` whenever possible.
I think the type of those props should be `string`